sale_layout_category_hide_detail/static/src/js/sale_layout_category_hide_detail.js

```diff
@@ -37,16 +39,25 @@ odoo.define('sale_layout_category_hide_detail.sale_layout_category_hide_detail',
             var section_fields_count = 0;
             var self = this;
             this.columns.forEach(function(elem) {
-                var options = self.state.fieldsInfo.list[elem.attrs.name].options;
-                if (options.show_in_line_section)
+
+                var show_in_line_section = false;
+                var field_info = self.state.fieldsInfo.list[elem.attrs.name];
+                if (field_info)
+                    show_in_line_section = field_info.options.show_in_line_section;
+
+                if (show_in_line_section)
                     section_fields_count ++;
             });
             return section_fields_count;
         },
         _renderHeaderCell: function (node) {
             var $th = this._super.apply(this, arguments);
-            var options = this.state.fieldsInfo.list[node.attrs.name].options;
-            var show_in_line_section = options.show_in_line_section;
+
+            var show_in_line_section = false;
```

Comment:
@Tardo can you help in a more compact form of putting this?

Reply:
Something like this?
```
var field_info = this.state.fieldsInfo.list[node.attrs.name];
var show_in_line_section = field_info && field_info.options.show_in_line_section;
$th.text("").toggleClass('o_column_sortable', !show_in_line_section);
```

Reply:
Uhm, not sure. `o_column_sortable` appears in the rest of the cases, @ernestotejeda ?

Reply:
Yes, I just want to remove o_column_sortable class  in case show_in_line_section is true